scripts/config

```diff
@@ -71,6 +73,26 @@ if __name__ == "__main__":
     with open(config_file, encoding='utf-8') as data_file:
         config = yaml.load(data_file.read())
 
+    all_tags = []
+    platform_build_url = "https://android.googlesource.com/platform/build"
+    tag_filter = "android-{}".format(config['version'])
+    process = subprocess.run(["git", "ls-remote", "--refs", "--tags", "--sort=v:refname", platform_build_url], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Any reason to not use the list-remote in the python git library like we did here?

https://github.com/hashbang/aosp-build/blob/6eb449e0296e1d3f2e6739ed759541d521f85541/scripts/manifest#L4

Reply:
https://github.com/hashbang/aosp-build/blob/6eb449e0296e1d3f2e6739ed759541d521f85541/scripts/manifest#L144-L150